notebook/data_exploration/dataset_stats.ipynb

```diff
@@ -0,0 +1,262 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Dataset Statistics\n",
+    "This notebook keeps track of various statistics about the compendium dataset like how many samples are labled, which diseases are present, etc.\n",
+    "\n",
+    "To skip straight to the stats, click [here](#stats)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from collections import Counter\n",
+    "import json\n",
+    "import os\n",
+    "import sys\n",
+    "\n",
+    "# __file__ isn't defined for jupyter notebooks\n",
+    "curr_path = os.path.dirname(os.path.abspath(os.path.abspath('')))\n",
+    "whistl_path = os.path.join(curr_path, os.pardir, 'whistl')\n",
+    "sys.path.append(whistl_path)\n",
+    "import utils"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "map_file = os.path.join(curr_path, os.pardir, 'data', 'sample_classifications.pkl')\n",
+    "sample_to_label = utils.parse_map_file(map_file)\n",
+    "sample_ids = sample_to_label.keys()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Not all labeled samples show up in the compendium, which causes pandas to panic. To fix this we have to \n",
+    "# take the intersection of the accessions in sample_ids and the accessions in the compendium\n",
+    "compendium_path = os.path.join(curr_path, os.pardir, 'data', 'human_compendium', 'HOMO_SAPIENS.tsv')\n",
+    "\n",
+    "header_ids = None\n",
+    "with open(compendium_path) as in_file:\n",
+    "    header = in_file.readline()\n",
+    "    header_ids = header.split('\\t')\n",
+    "\n",
+    "valid_sample_ids = [id_ for id_ in sample_ids if id_ in header_ids]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "metadata_file = os.path.join(curr_path, os.pardir, 'data', 'all_metadata.json')\n",
```

Comment:
Maybe add some more comments in cells to explain what you're doing for an easier read the next time you refer back to these scripts/for helping to debug anything in the future